specification/cognitiveservices/data-plane/Language/preview/2022-04-01-preview/textanalytics.json

```diff
@@ -1966,6 +2559,30 @@
       "type": "array",
       "uniqueItems": true
     },
+    "TranscriptionPiiCategories": {
+      "description": "(Optional) describes the PII categories to return for transcription conversation detection.",
+      "items": {
+        "type": "string",
+        "x-ms-enum": {
+          "name": "TranscriptionPiiCategory",
+          "modelAsString": true
+        },
+        "enum": [
+          "Address",
+          "CreditCardNumber",
+          "Email",
+          "Name",
+          "NumericIdentifier",
+          "PhoneNumber",
+          "USSocialSecurityNumber",
```

Comment:
Not that this should be the default for `En` only.

Reply:
As noted above, too, can we also document what all of these are? In these cases, showing an example would be helpful documentation. E.g.,

```
A Social Security Number in the USA e.g., "123-45-6789".
```

Reply:
Will add.